src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/CcdAuthInfo.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services;
+
+import uk.gov.hmcts.reform.idam.client.models.UserDetails;
+
+public class CcdAuthInfo {
+    final UserDetails userDetails;
+    final String serviceToken;
```

Comment:
we shouldn't be passing this around, we should 'generate' one each time, 
Its cached and will either return a valid one or will generate a new one if required
Otherwise the token could expire in between calls